ubuntudesign/www.ubuntu.com
# Cu bg and code

## Done

Added information and videos about conjure-up to ubuntu.com
 - updates to /cloud, /cloud/openstack and /download/cloud
 - new /download/cloud/autopilot and /download/cloud/conjure-up pages

## QA

All code in this has been previously PR'd into the feature branch. Sanity check and make sure that this is a clean merge.
